src/resources/Microsoft.Authorization/policyDefinitions/Append-AppService-latestTLS.json

```diff
@@ -11,7 +11,12 @@
     "metadata": {
       "version": "1.0.0",
       "category": "App Service",
-      "source": "https://github.com/Azure/Enterprise-Scale/"
+      "source": "https://github.com/Azure/Enterprise-Scale/",
+      "alzCloudEnvironments": [
+        "AzureCloud",
+        "AzureChinaCloud",
+        "AzureUSGovernment"
+      ]
```

Comment:
Do we need to remove this @krowlandson as @jfaurskov is handling this in another PR to be created?

This PR should just be the workflow update right?

Reply:
Yes... this was just to test that the updated workflow worked. Should be removed now.